drivers/ethernet/eth_xlnx_gem_priv.h

```diff
@@ -385,8 +385,8 @@
 #define ETH_XLNX_GEM_IXR_RX_USED_BIT			0x00000004
 #define ETH_XLNX_GEM_IXR_FRAME_RX_BIT			0x00000002
 #define ETH_XLNX_GEM_IXR_PHY_MGMNT_BIT			0x00000001
-#define ETH_XLNX_GEM_IXR_ALL_MASK			0x03FC7FFE
-#define ETH_XLNX_GEM_IXR_ERRORS_MASK			0x00000C60
+#define ETH_XLNX_GEM_IXR_ALL_RELEVANT_MASK		0x00000CF2
+#define ETH_XLNX_GEM_IXR_ERRORS_MASK			0x00000C70
```

Comment:
I looked into the question regarding the TX underrun bit again, I most likely didn't include it because the UltraScale has it but the Zynq doesn't. So this bit will be a SoC-#ifdef'd inclusion into the interrupt source bit mask.